kivy/uix/video.py

```diff
@@ -163,6 +213,9 @@ def seek(self, percent, precise=True):
             raise Exception('Video not loaded.')
         self._video.seek(percent, precise=precise)
 
+    def _update_source_from_preview(self, inst, val):
+        self.source = self.preview_source
```

Comment:
I fear we introduce an unwanted behavior change here.

in my initial implementation it was possible to set ``source`` and ``video_source`` on an instane of ``Video`` and both triggered a video load. Now, setting ``source`` has no effect any mode (L135) but instead we cross-map ``source`` and ``preview_source``. Thats what i wanted to avoid in the first place. ``preview_source`` is useful only on widget ``__init__`` in the initial implementation (by intent)

Reply:
To be honest, i also not see why keeping ``video_source`` and ``source`` not in sync is a bad thing. The ``preview_image`` in it's current implementation is meant for initial placeholder only when the video gets instanciated to prevent the white canvas while loading (or as long as the video not gets started).

An additional case would be to display this image again once the player stops, but that would require acting on eos and is another aspect.

All in all it feels to me that the initial implementation is less immersive